google/cloud/aiplatform/base.py

```diff
@@ -541,6 +541,14 @@ def update_time(self) -> datetime.datetime:
     def __repr__(self) -> str:
         return f"{object.__repr__(self)} \nresource name: {self.resource_name}"
 
+    def __getattr__(self, attr) -> Any:
```

Comment:
Since this is an exposed API, I would consider removing this and, instead, explicitly add attributes with documentation. A possible drop in replacement is a `gca_resource` property, perhaps with a more descriptive name.

Reply:
Dropping after further discussion, we'll stick to exposing class properties individually